jp/components/chip.md

```diff
@@ -166,13 +166,22 @@ public chipsOrderChanged(event) {
   - <kbd>LEFT</kbd> - 左側にあるチップをフォーカスします。
   - <kbd>RIGHT</kbd> - 右側にあるチップをフォーカスします。
   - <kbd>SPACE</kbd> - チップが選択可能な場合、選択状態を切り替えます。
-  - <kbd>DELETE</kbd> - チップの削除を手動的に処理するために `onRemove` 出力を発生します。
+  - <kbd>DELETE</kbd> - チップの削除を手動的に処理するために [`onRemove`]({environment:angularApiUrl}/classes/igxchipcomponent.html#onremove) 出力を発生します。
   - <kbd>SHIFT</kbd> + <kbd>LEFT</kbd> - フォーカスされたチップの位置を 1 つ左へ移動します。
   - <kbd>SHIFT</kbd> + <kbd>RIGHT</kbd> - フォーカスされたチップの位置を 1 つ右へ移動します。
 
 - 削除ボタンがフォーカスされた場合のキーボード コントロール:
 
-  - <kbd>SPACE</kbd> または <kbd>ENTER</kbd> チップの削除を手動的に処理するために `onRemove` 出力を発生します。
+  - <kbd>SPACE</kbd> または <kbd>ENTER</kbd> チップの削除を手動的に処理するために [`onRemove`]({environment:angularApiUrl}/classes/igxchipcomponent.html#onremove) 出力を発生します。
+
+### API
+
+* [IgxChipComponent]({environment:angularApiUrl}/classes/igxchipcomponent.html)
```

Comment:
Provide Sassdoc reference to the igx-chip-theme.

Reply:
Please review the changes that were made.